OverPassLayer.js

```diff
@@ -229,6 +244,11 @@ L.OverPassLayer = L.FeatureGroup.extend({
           var queryWithMapCoordinates = this.options.query.replace(/(BBOX)/g, bbox.toOverpassBBoxString());
           var url =  this.options.endpoint + "interpreter?data=[out:json];" + queryWithMapCoordinates;
 
+          if (beforeRequest) {
```

Comment:
why do you check for `beforeRequest` ? As far as I understand the code - every time you reach this point, it's before the next request.